WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -360,4 +361,19 @@ private boolean mediaAlreadyQueuedOrUploading(MediaModel mediaModel) {
         }
         return false;
     }
-}
\ No newline at end of file
+
+    public static boolean hasPendingMediaUploadsForPost(PostModel postModel) {
```

Comment:
I don't think we should - a style rule of using FluxC is to always use models wherever possible. This avoids ambiguities we used to have where you get passed an `id` that could represent either a local id or a remote id.